tests/foreman/api/test_rhcloud_inventory.py

```diff
@@ -154,8 +154,8 @@ def test_rhcloud_inventory_api_hosts_synchronization(
     task_output = module_target_sat.api.ForemanTask().search(
         query={'search': f'id = {inventory_sync["task"]["id"]}'}
     )
-    assert task_output[0].output['host_statuses']['sync'] == 2
-    assert task_output[0].output['host_statuses']['disconnect'] == 0
```

Comment:
@ColeHiggins2 this is not good. If the assertions are failing that means there's a possibility of a bug in the product here. You should raise this with devs.

Reply:
Seems like this is saying there should be 2 hosts synced and 0 disconnected. I agree that this looks like the product is not working and the assertions should not change.